src/serpentTools/parsers/depmatrix.py

```diff
@@ -41,6 +41,8 @@ class DepmtxReader(BaseReader, SparseReaderMixin):
     ----------
     deltaT: float
         Length of depletion interval
+    flx: float
+        Homogenized flux
```

Comment:
Can you add a comment about this not existing for versions of Serpent less than 2.2.1